demos/ASP.Net MVC/Divergent.Frontend/Views/Orders/Details.cshtml

```diff
@@ -16,12 +15,3 @@
         <div>Courier: @Model.ShippingCourier</div>
     </div>
 </div>
-
-@*<div class="panel panel-default">
-    <div class="panel-heading">
-        <h3 class="panel-title">Shipment information</h3>
-    </div>
-    <div class="panel-body">
-
-    </div>
-</div>*@
```

Comment:
@mauroservienti I would like to revert the changes to these files:

- demos/ASP.Net MVC/.vs/ASP.Net MVC/v14/.suo
- demos/ASP.Net MVC/Divergent.Frontend/Views/Orders/Details.cshtml

I don't think a specific commit needs to be reverted. They can both be reverted to what's in `master`.